testing/src/test/java/org/motechproject/nms/testing/it/kilkari/MctsBeneficiaryUpdateServiceBundleIT.java

```diff
@@ -521,7 +521,7 @@ public void testUpdateBeneficiariesFromFile() throws Exception {
         assertEquals(453L, (long) updatedMother3.getHealthBlock().getCode());
 
         // DOB update:
-        String updatedDOB = "01-07-2015";
+        String updatedDOB = "01-12-2015";
```

Comment:
yup, these keep constantly rolling off. we either have it in a file or manually write the string to a stream.